coreos-4/policies/RA-Risk_Assessment/component.yaml

```diff
@@ -149,12 +149,11 @@
 - control_key: RA-5 (1)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: not applicable
   narrative:
     - text: |
-        A control response is planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-353
+        Organizational policy and procedures to employ and update the organizational
+        scanning appliance is outside the scope of Red Hat CoreOS configuration.
```

Comment:
```suggestion
        Organizational policies and procedures to employ and an organizational
        scanning tool are outside the scope of Red Hat CoreOS configuration.
```

Reply:
In the spirit of getting /something/ out the door, made a few minor tweaks. Ultimately a complete control response will document the native tools CoreOS delivers for vulnerability scanning.

Reply:
I agree - but how are we defining native? Anything available through the configured BaseOS, AppStream and EPEL repos? That seems to broaden the scope of potential audit questions immensely. It seems that the only actually native scanner for the COS/OCP stack will be the security operator that is currently in-progress.